autosubmit/autosubmit.py

```diff
@@ -1368,8 +1370,8 @@ def expid(description, hpc="", copy_id='', dummy=False, minimal_configuration=Fa
         try:
             # Copy another experiment from the database
             if copy_id != '' and copy_id is not None:
-                copy_id_folder = os.path.join(root_folder, copy_id)
-                if not os.path.exists(copy_id_folder):
+                copy_id_folder = root_folder / copy_id
+                if not copy_id_folder.exists():
```

Comment:
I think these two are not being tested. Would be useful if this can be reached with tests (unit/integration).